src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,344 @@
+import type p5 from 'p5'
+import {VisualizerDefault} from './VisualizerDefault'
+import type {VisualizerInterface} from '@/visualizers/VisualizerInterface'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import type {SequenceInterface} from '../sequences/SequenceInterface'
+
+/** md
+# Sequence Color Visualizer
+In this visualizer, the brightness of each circle represents how fast $1/a^n$, 
+where a is a number in the sequence, 
+approaches zero as n increases.
+Each prime number is assigned a color, 
+and the color of each circle represents 
+the color combination of its prime factors. \s
+
+Press the "Left arrow key" to show/hide the color of each prime factor.\s
+Press the "Right arrow key" to show/hide labels.\s
+Press "[" to turn up brightness.\s
+Press "]" to turn down brightness. \s
+ **/
+
+const colorMap = new Map()
+
+// Show description box by pressing left arrow key
+class SeqColor extends VisualizerDefault implements VisualizerInterface {
+    name = 'Sequence Color'
+    n = 40
+
+    params = {
+        n: {
+            value: this.n,
+            forceType: 'integer',
+            displayName: 'number of terms',
+            required: true,
+        },
+    }
+
+    private currentNum = 0
+    private X = 0
+    private Y = 0
```

Comment:
When I submitted my Chaos Visualizer for a PR, @gwhitney encouraged me to use the p5.Vector class which has attributes x and y. See the chaos visualizer code [here](https://github.com/numberscope/frontscope/blob/bb157252d6ce8dfb603bbb0860531daee34a3053/src/visualizers/VisualizerChaos.ts#L236). That might be nice for some of your x and y coordinate variables.

Reply:
done; check it out

Reply:
Vectors seem mostly appropriately used; resolving.